blackjax/hmc.py

```diff
@@ -0,0 +1,101 @@
+"""Public API for the HMC Kernel"""
+from typing import Callable, Dict, List, NamedTuple, Tuple, Union
+
+import jax
+import jax.numpy as jnp
+import numpy as np
+
+import blackjax.inference.base as base
+import blackjax.inference.integrators as integrators
+import blackjax.inference.metrics as metrics
+import blackjax.inference.proposals as proposals
+
+Array = Union[np.array, jnp.DeviceArray]
+PyTree = Union[Dict, List, Tuple]
```

Comment:
Those types are repeated, probably nicer to have them in single place for consistency.

Reply:
I agree. I usually duplicate things until where that single place should be becomes obvious.